docs/docs/adding-search.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Adding search to your Gatsby website"
+---
+
+There are few ways to approach adding search to your Gatsby-powered site:
+
+1. You can use libraries such as [elesticlunr](https://www.npmjs.com/package/elasticlunr) for offline search but doing so will require you to index at build time. Fortunately, this is achievable using the [gatsby-plugin-elasticlunr-search](https://github.com/andrew-codes/gatsby-plugin-elasticlunr-search) plugin.
```

Comment:
You might also want to add a mention for js-search as well  that the entire search index has to be brought into the client — I've tested this and for sites even around ~100 posts — the search index can get > 250kb (my memory of the numbers is a bit vague but it goes up quickly). You can offset this by only indexing titles and such but it's tricky for all except small sites.